config.toml

```diff
@@ -4,10 +4,14 @@ theme = "startwords"
 copyright = "This work is licensed under a Creative Commons Attribution 4.0 International License."
 enableEmoji = true
 googleAnalytics = "UA-87887700-8"
+enableRobotsTXT = true
+enableGitInfo = true
```

Comment:
what's the `GitInfo` do?

Reply:
It uses git commit info for generating last modified dates for content — see https://gohugo.io/variables/git/

Reply:
Should we add a comment in the config? 

From what I read this makes the build a little slower, but seems worth it to me. The github actions build is based on a git checkout, so I think it should work.

Reply:
cool! seems worth it to me too. a comment wouldn't hurt.